cookbooks/prescribed_velocity/corner_flow.prm

```diff
@@ -5,6 +5,11 @@ set End time    = 0
 # Load the signal library.
 set Additional shared libraries = ./libprescribed_velocity.so
 
+subsection Discretization
```

Comment:
Ah, I see. But why change the cookbook?

As for test: Yes, that would be very useful indeed -- we usually try to exercise all code paths in the testsuite somehow.